### PR TITLE
Dact 2441 address formatting country and postcode

### PR DESCRIPTION
**Pull request by olaych**
Jira: https://companieshouse.atlassian.net/browse/DACT-2441

### CODE REVIEW

File: src/utils/format.ts [hunk 1]
```diff
@@ -146,3 +147,23 @@ export const formatNationalitiesToSentenceCase = (nationality: string | undefine
     return `${word.slice(0, 1)}${word.slice(1).toLowerCase()}`;
   });
 };
+
+export const formatAddress = (addressFields: (string | undefined)[]): string => {
+  return addressFields
+    .filter(Boolean)
```
[the remark this answers is not in the document]
Comment (AndreKramer7654) [reply src/utils/format.ts:153]: after map?

[the remark this answers is not in the document]
Comment (olaych) [reply src/utils/format.ts]: Updated

File: src/utils/format.ts [hunk 2]
```diff
@@ -158,3 +159,22 @@ export const formatNationalitiesToSentenceCase = (nationality: string | undefine
   });
 };
 
+export const formatAddress = (addressFields: (string | undefined)[]): string => {
+  return addressFields
+    .filter(field => field != undefined && field !== "")
```
Comment (AndreKramer7654) [inline src/utils/format.ts:164]: .map(s => s?.trim())
.filter(field => field !== undefined && field !== "")
.join(',  ')

Note !== instead of !=

Comment (olaych) [reply src/utils/format.ts]: Fixed

**Review by AndreKramer7654 — APPROVED** [review]

**Review by PhilipGlover — APPROVED** [review]
👍

**Pull request merged**